examples/apps/Django_Panel_Multi/Django_Panel_Multi/themes.py

```diff
@@ -0,0 +1,21 @@
+import holoviews as hv
+import panel as pn
+
+
+def __disable_logo(plot, element):
+    plot.state.toolbar.logo = None
+
+
+def plot_themes():
+    # format
+    hv.plotting.bokeh.ElementPlot.finalize_hooks.append(__disable_logo)
+    pn.widgets.DatetimeInput.format = '%d %B %Y'
+    hv.plotting.bokeh.ElementPlot.bgcolor = "#fbfcfc"
+    hv.plotting.bokeh.ElementPlot.gridstyle = {"grid_line_alpha": 0.6, "grid_line_dash": 'dashed'}
+    hv.Palette.colormaps = 'Blues'
+
+    # Reminder
+    # legend positions
+    # ['top_right', 'top_left', 'bottom_left', 'bottom_right']
+    # ['right', 'left', 'top', 'bottom']
+    # return dfp.hvplot(grid=True, colormap='Paired').opts(legend_position='top_left')
```

Comment:
Let's clean this out before we merge.

Reply:
Yes...not in a merging state yet..probably next week